providers/curation/github.js

```diff
@@ -109,13 +110,43 @@ class GitHubCurationService {
     // Github requires name/email to set committer
     if ((info.name || info.login) && info.email)
       fileBody.committer = { name: info.name || info.login, email: info.email }
+    else logger.info('Github requires name/email to set committer, info.email is currently: ', info.email)
     if (get(currentContent, '_origin.sha')) {
       fileBody.sha = currentContent._origin.sha
+      // Extract the component values
+      const { coordinates, revisions } = currentContent
+      const { name, provider, type } = coordinates
+      // Only get the latest revision
+
+      let newRevision = Object.keys(patch.revisions)[0]
```

Comment:
I'm torn on this. We can check that all versions exist or just that the component exists. Feels like in the end we should be checking that all things mentioned in the proposed patch exists.